core-java/src/test/java/com/baeldung/utils/UtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.utils;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UtilsTest {
+
+        @Test public void givenString_whenCallingStringToCharArray_shouldConvertToCharArray() {
+                String givenString = "abcd";
+
+                char[] result = Utils.stringToCharArray(givenString);
+
+                assertTrue(Arrays.equals(result, new char[] { 'a', 'b', 'c', 'd' }));
+        }
+
+        @Test public void givenCharArray_whenCallingCharArrayToString_shouldConvertToString() {
```

Comment:
Did you use the default formatter?  The annotation in the same line as the method is a convention that we do not follow